lib/routes/account.js

```diff
@@ -659,6 +659,14 @@ module.exports = function (
               log.event('verified', { email: account.email, uid: account.uid, locale: account.locale })
               log.increment('account.verified')
               return db.verifyEmail(account)
+                .then(mailer.sendPostVerifyEmail.bind(
+                    mailer,
+                    account.email,
+                    {
+                      acceptLanguage: request.app.acceptLanguage
+                    }
+                  )
+                )
```

Comment:
Actually, the email copy for this says "You successfully connected your first device to your Firefox Account".  Maybe we should only send this if `service=sync`?  It doesn't seem appropriate if they e.g. created the account in order to log in to Hello or Pocket.